README.md

```diff
@@ -1,3 +1,32 @@
 # loopback-next-extension-starter
 
 A starter project to create extensions for loopback-next.
+
```

Comment:
Missing `## Overview` section header

Reply:
I'll be updating the top level README once all the different extensions have landed